iotdb/src/main/java/org/apache/iotdb/db/engine/filenode/FileNodeManager.java

```diff
@@ -897,6 +899,29 @@ public void deleteOneFileNode(String processorName) throws FileNodeManagerExcept
     }
   }
 
+  private void cleanOverflow(String processorName) throws IOException {
+    String overflowDirPath = TsFileDBConf.getOverflowDataDir();
+    String overflowPath = standardizeDir(overflowDirPath) + processorName;
+    File overflowDir = new File(overflowPath);
+    // free and close the streams under this overflow directory
+    if (!overflowDir.exists()) {
+      return;
+    }
+    File[] overflowSubDirs = overflowDir.listFiles();
+    if (overflowSubDirs == null) {
+      return;
+    }
+    for (File overflowSubDir : overflowSubDirs) {
+      File[] overflowFiles = overflowSubDir.listFiles();
```

Comment:
line 915-920 are duplicated with 934-939

Reply:
Is a duplication of 5 lines intolerable?